pkg/ebpftracer/c/headers/common/network.h

```diff
@@ -460,7 +460,7 @@ statfunc int get_remote_sockaddr_in6_from_network_details(struct sockaddr_in6 *a
     return 0;
 }
 
-statfunc bool fill_tuple(struct sock *sk, tuple_t *tuple)
+statfunc bool fill_tuple_from_sock(struct sock *sk, tuple_t *tuple)
```

Comment:
Is this function used? I tried searching for it, but couldn't find anything in the PR